themes-default/slim/gulpfile.js

```diff
@@ -20,14 +19,18 @@ const argv = require('yargs').argv;
 const rename = require('gulp-rename');
 const changed = require('gulp-changed');
 
+const xo = require('xo');
+
+const xoReporter = xo.getFormatter('eslint-formatter-pretty');
+
 // Const postcss = require('gulp-postcss');
 // const sass = require('gulp-sass');
 const cssnano = require('cssnano');
 const autoprefixer = require('autoprefixer');
 const reporter = require('postcss-reporter');
 
 const PROD = process.env.NODE_ENV === 'production';
-const config = require('./package.json').config;
+const { config, xo: xoConfig } = require('./package.json');
```

Comment:
Didn't know you could rename a deconstructed param.

Reply:
You and me both. That's pretty neat. Hopefully it's readable enough...